jsx/App/Stories/Story/Display/Untimed.jsx

```diff
@@ -1,17 +1,37 @@
 import id from 'shortid';
 import { Sentence } from "./Sentence.jsx";
 
+function UntimedBlock({ sentence, sentenceId }) {
+	return (
+		<div className="untimedBlock">
+			<span className="timeStampContainer timeStamp" data-sentence_id={sentenceId}>
+				{sentenceId}
+			</span>
+			<Sentence sentence={sentence} />
+		</div>
+	);
+}
+
 export function UntimedTextDisplay({ sentences }) {
 	// I/P: sentences, a list of sentences
 	// O/P: the main gloss view, with several Sentences arranged vertically, each wrapped in an UntimedBlock
 	// Status: tested, working
 	let output = [];
+	let sentenceCount = 1; 
 	for (const sentence of sentences) {
 		output.push(
-			<div key={id.generate()} className="untimedBlock">
-				<Sentence key={id.generate()} sentence={sentence} />
-			</div>
+			<UntimedBlock 
+				key={id.generate()} 
+				sentence={sentence} 
+				sentenceId={sentenceCount} 
+			/>
+			
+			// ----- Original code -----
```

Comment:
Let's delete these commented-out lines, unless you think they'll add to readability in the future. The old version will always be available in the git history!

Reply:
Updated